examples/vision/vit_small_ds.py

```diff
@@ -34,30 +34,34 @@
 example is inspired from
 [Image classification with Vision Transformer](https://keras.io/examples/vision/image_classification_with_vision_transformer/).
 
-_Note_: This example requires TensorFlow 2.6 or higher, as well as
-[TensorFlow Addons](https://www.tensorflow.org/addons), which can be
-installed using the following command:
+_Note_: This example requires TensorFlow 2.6 or higher
```

Comment:
You can remove this note since it is now inaccurate.

Reply:
Ok

Reply:
Changes done